backend/.env.example

```
@@ -1,4 +1,5 @@
 BOT_TOKEN=your_telegram_bot_token # Токен вашего чат-бота в Telegram
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)